pkg/controller/plan/kubevirt.go

```diff
@@ -834,6 +834,56 @@ func (r *KubeVirt) EnsureGuestConversionPod(vm *plan.VMStatus, vmCr *VirtualMach
 	return
 }
 
+func (r *KubeVirt) EnsureVirtV2VPVCStatus() (ready bool, err error) {
+	pvcs := &core.PersistentVolumeClaimList{}
+	pvcLabels := map[string]string{
+		"migration": r.Migration.Name,
+		"ova":       "nfs-pvc",
```

Comment:
`nfs-pvc` is used 3 times (including this change) in this file. how about making it a const?